src/components/ModalDialog/ModalDialog.js

```diff
@@ -61,13 +61,31 @@ class ModalDialog extends Component {
     useNotificationFormat: false
   }
 
+  modalRef = createRef()
+
+  handleKeydown = event => {
+    if (event.code === 'Escape') this.submit()
+    if (event.code === 'Enter') this.cancel()
```

Comment:
and this submit?

Reply:
Ooh thanks ... yes, I recall that I originally was using event.keyCode and then switches to event.code ... and didn't switch the calls. Commit pushed